### DISCUSSION TITLE
+Project name

### DISCUSSION THREAD

**Discussion by ghost** [porsager/postgres#612] [General]
No doubt it's not changeable now, but it would have been so much better if this project had a unique name instead of "Postgres".

It makes it effectively impossible to search for information on this project.

**Comment by porsager** [comment]
I try to refer to it as Postgres.js, so you can try that 😉 

Still shouldn't need more information than the readme - what are you missing? 😊

**Comment by ghost** [comment]
The repo is called "postgres"

For example:
npm install postgres

import postgres from 'postgres'

const sql = postgres()

It's pretty easy to believe this project is named "postgres".

Information about your project will end up on Twitter, StackOverflow, Google, ChatGPT, blogs, tutorials.

Without a unique name, your project effectively is invisible.  Searching for "postgres" on google turns up lots of results for postgres, but not for your project.  That means people who might love to use it don't discover it, and people who need help solving problems and using it will find it hard to get help.

The more unique the name, the better it is for problem solving, and also for the popularity of your project.

"slonik" or "sequelize" are very distinctive names - search for them and you'll get relevant information.

A project that named postgres or maybe postgres.js is very bad for people who want to find things about the project.

In this space there is pg node-postgres postgres (or maybe postgres.js) pg-protocol pg-hstore and goodness knows how many others all very similar. I can't recall any of them distinctively by name.  However, slonik and sequelize instantly come to mind and are highly searchable.

It's your project, and it's a good one so of course it's up to you, I'm just saying that you do your users and potential users a favor when you give your project a nice unique, searchable name.

**Comment by porsager** [comment]
> That means people who might love to use it don't discover it, and people who need help solving problems and using it will find it hard to get help.

It requires that the project is already known by the ones searching for it. 

I think these two search results show that the choice of name was pretty good 😊 

https://www.google.dk/search?q=Postgres+js&ie=UTF-8&oe=UTF-8&hl=en-dk&client=safari

https://www.npmjs.com/search?q=postgres

I don't think so many people would have discovered it, had I kept the name "peegee" I used while making the initial POC. 

As for finding information other places than here, or perhaps a future homepage, I'm still not sure what the benefit of that is? What information are you missing?
